File: app/src/main/java/com/example/helthlv/MissionSucces.java
package com.example.helthlv;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class MissionSucces extends AppCompatActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_succes);

        Button btn11 = findViewById(R.id.btn11);
        btn11.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/health_project/mission/MissionSucces.kt
package com.example.health_project.mission

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.health_project.R
import android.content.Intent
import android.widget.Button
import com.example.health_project.MainActivity

class MissionSucces : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_succes)
        val btn11 = findViewById<Button>(R.id.btn11)
        btn11.setOnClickListener {
            val intent = Intent(applicationContext, MainActivity::class.java)
            startActivity(intent)
        }
    }
}